webapp-src/src/Login/Buttons.js

```diff
@@ -140,29 +142,31 @@ class Buttons extends Component {
         </div>
       </div>;
   		return (
-        <div>
+        <div class="text-center">
           <div className="btn-group" role="group">
             {bContinue}
-            <button type="button" className="btn btn-primary" onClick={this.clickLogout}>
+            <button type="button" className="btn btn-danger" onClick={this.clickLogout}>
```

Comment:
I was also wondering if having a green button and a red button right next to each other is a good idea, the contrast is very high.

I tried using other bootstrap classes or separating the buttons but I wouldn't say it's better.
![Capture d’écran du 2020-06-14 20-53-09](https://user-images.githubusercontent.com/3966617/84608589-26038300-ae81-11ea-8ba0-afc378cc6507.png)
I'll try it for a few and I'll probably get used to it. :-)

Reply:
I tried the "warning" style (orange) for logout but it seemed unconvincing. Custom button colors are always a possibility, of course. Thanks for adding padding between the two buttons, as I see in your screenshot.

Reply:
(By the way, it is always possible to use the `btn-secondary` (gray) class for the "Logout" button, as well. Its privileged position next to "Continue" might bring it out just enough—but not too much. I don't have a strong opinion about this. I'd never wear a green t-shirt with red trousers—or anything with red trousers—but with buttons on a very temporary webpage I can find the contrast pleasing.)